CONTRIBUTING.md

```diff
@@ -1,44 +1,41 @@
 # Contributors guidelines
 
 This document summarizes the most important points for people interested in
-contributing to Godot, especially via bug reports or pull requests.
+contributing to Blazium, especially via bug reports or pull requests.
 
-The Godot documentation has a dedicated [Contributing section](https://docs.godotengine.org/en/latest/contributing/how_to_contribute.html)
+The [Blazium Discord](https://discord.gg/redot) documentation has a dedicated Contributing section
 which details these points and more, and is a recommended read.
 
 ## Table of contents
 
 - [Reporting bugs](#reporting-bugs)
 - [Proposing features or improvements](#proposing-features-or-improvements)
 - [Contributing pull requests](#contributing-pull-requests)
-- [Contributing to Godot translations](#contributing-to-godot-translations)
+- [Contributing to Blazium translations](#contributing-to-blazium-translations)
 - [Communicating with developers](#communicating-with-developers)
 
 ## Reporting bugs
 
-Report bugs [here](https://github.com/godotengine/godot/issues/new?assignees=&labels=&template=bug_report.yml).
+Report bugs [here](https://github.com/blazium-engine/blazium/issues/new?assignees=&labels=&template=bug_report.yml).
 Please follow the instructions in the template when you do.
 
 Notably, please include a Minimal Reproduction Project (MRP), which is a small
-Godot project which reproduces the issue, with no unnecessary files included.
-Be sure to not include the `.godot` folder in the archive to save space.
+Blazium project which reproduces the issue, with no unnecessary files included.
+Be sure to not include the `.blazium` folder in the archive to save space.
 
-Make sure that the bug you are experiencing is reproducible in the latest Godot
-releases. You can find an overview of all Godot releases [on the website](https://godotengine.org/download/archive/)
-to confirm whether your current version is the latest one. It's worth testing
-against both the latest stable release and the latest dev snapshot for the next
-Godot release.
+Make sure that the bug you are experiencing is reproducible in the latest Blazium
+releases. It's worth testing against both the latest stable release and the
+latest dev snapshot for the next Blazium release.
 
-If you run into a bug which wasn't present in an earlier Godot version (what we
+If you run into a bug which wasn't present in an earlier Blazium version (what we
 call a _regression_), please mention it and clarify which versions you tested
 (both the one(s) working and the one(s) exhibiting the bug).
 
 ## Proposing features or improvements
 
 **The main issue tracker is for bug reports and does not accept feature proposals.**
 
-Instead, head to the [Godot Proposals repository](https://github.com/godotengine/godot-proposals)
-and follow the instructions in the README file and issue template.
+Instead, head to the [Blazium Discord](https://discord.gg/redot).
```

Comment:
This still has redot in it.

Reply:
Hello! 

One question we had was if we could include the discord link as of right now (Since it's not open yet).

Reply:
Work completed